### ISSUE TITLE
False positives on ETSY006

### ISSUE THREAD

**Issue by bjozet** [etsy/foodcritic-rules#14]
I get false positives for ETSY006 when using "execute" resources on a non-core binary.

For example, I have:

```
    execute "/usr/sbin/vmtouch -d -L /my/path" do
      not_if "pidof vmtouch"
    end
```

"touch" is considered to be core, "vmtouch" is not, but still matches.
